blocks/form/submit.js

```diff
@@ -136,7 +136,8 @@ async function constructPayload(form, captcha) {
     ':currentPagePath': '/content/shred-it/us/en',
     currentPagePath: window.location.pathname,
     jobPropertiesUrl: `https://main--shredit--stericycle.aem.page${form.dataset.action}.json`,
-    formURL: form.dataset?.action,
+    formName: form.dataset?.action,
+    formURL: window.location.href,
```

Comment:
actual its page URL rite not form URL?

Reply:
As per colin formURL should be pageURL that's the expectation, see issue #817